src/software/guides/addin-storage.md

```diff
@@ -0,0 +1,384 @@
+
+The AddInData GUID is a mandatory parameter when calling the AddInData method.
+ This allows each Add-In&#39;s data to be isolated from the data used by other Add-Ins.
+**Consider the AddInId GUID the "serial number" for your solution.**
+
+The AddInData GUID allows multiple solutions to each have their own collection of AddInData objects in the same database without the collections mixing. To generate your own GUID, please click on the following [example]({{site.baseurl}}/software/api/runner.html#sample:generate-addin-guid).
+
+## Creating an AddInData Object
+
+An AddInData object must first be created in a database. The properties of AddInData are as follows:
+
+| **Properties** |
+| --- |
+| [Id]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.Id): The standard Id for any Entity. |
+| [AddInId]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.Id): Used to identify the Add-In Solution to which this AddInData belongs. Add-Ins cannot see data from other Add-Ins. Must be provided when searching for AddInData. Consider this a Serial Number for this Solution that uses the Storage API. |
+| [Groups]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.Group): Used to define the scope of a row of Add-In data. Works the same as any other ObjectModel Entity. Refer to the MyGeotab SDK [What&#39;s New](https://geotab.github.io/sdk/resources/new/) page for more information regarding the transition of this property from being required to optional. |
+| Details (String): The JSON data. May be whole or partial depending on the action (Add vs. Set) or the filtering provided when calling Get.|
+
+As an example, you can use the [API Runner tool](https://my.geotab.com/sdk/api/apiRunner.html) to create an AddInData object using the following operation:
+
+```javascript
+api.call("Add",
+{
+  "typeName": "AddInData",
+  "entity": {
+    "addInId": "a2C4ABQuLFkepPVf6-4OKAQ",
+    "groups": [
+      {
+        "id": "GroupCompanyId"
+      }
+    ],
+    "details": "{\"date\":\"2016-01-01T00:00:00.000Z\",\"items\":[{\"name\":\"bottles\",\"price\":12},{\"name\":\"caps\",\"price\":20}],\"customer\":{\"name\":\"joesmith\",\"email\":\"joe@smith.com\"}}"
+  }
+});
+```
+
+**Important Notes**
+
+Each invocation of the Add operation will create a new AddInData object with a unique Id bound to the entered AddInData GUID. The Id of the AddInData object is required to remove the object with the **Remove** method. See below for an example.
+
+The value of the Data property is a JSON object converted to a JSON String or a string containing the JSON format with escaped quotes (\"). This JSON string does not support single quotes or newlines. A convenient way to create the escaped string from a JSON object is to use the [JSON.stringify()](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) method.
```

Comment:
This isn't true any more with `details` property

Reply:
Removed this line in 764501adf2514edddd8a2468e0a9123abd659036